AI/modules/signal/models/TCN/wrapper.py

```
@@ -0,0 +1,56 @@
+# AI/modules/signal/models/TCN/wrapper.py
+import torch
+import numpy as np
+import pandas as pd
+from AI.modules.signal.core.base_model import BaseSignalModel
+
+class TCNWrapper(BaseSignalModel):
+    """
+    TCN 모델 Wrapper: 단기 패턴 및 전환점 포착 엔진 [명세서 3번 준수]
+    """
+    def __init__(self, model_path=None, config=None):
```

Ruff (0.14.14)

[warning] Unused method argument: `model_path`

(ARG002)